### PR TITLE
Use human readable names in pom.xml for all task projects.

### LINKED ISSUE #172: Use Human readable Project Names.

### PR DESCRIPTION
**Pull request by cppwfs**
updated pom to include text for contributor agreement
Resolves #172

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -43,5 +43,13 @@ public class MyApp {
...
 ----
 
+=== Sign the Contributor License Agreement
+Before we accept a non-trivial patch or pull request we will need you to sign the
+https://cla.pivotal.io/sign/spring[Contributor License Agreement].
+Signing the contributor's agreement does not grant anyone commit rights to the main
+repository, but it does mean that we can accept your contributions, and you will get an
+author credit if we do.  Active contributors might be asked to join the core team, and
+given the ability to merge pull requests.
+
```
Comment (mminella) [inline README.adoc]: Can we break this out into a separate commit?  The addition of the CLA tool is more than just the documentation related to it...

Comment (cppwfs) [inline README.adoc]: sure.   But our project should have been migrated over to using the CLA tool.

**Comment by ghillert** [comment]
Merged with commit 21d4a1759191e5217d610e09b640438e8103a8fc
